### ISSUE TITLE
<br> tags being added to truncated lines

### ISSUE THREAD

**Issue by macsplan** [One-com/react-truncate#112]
Hi, any way of turning off the generation of <br> tags inside the truncated text?

**Comment by Lemaro86** [comment]
Yes, I have such issue too. And it break all view. I resolve it by br: display none, but it isn’t good. Need add space for after span - bad hack

**Comment by Jaikant** [comment]
+1 Same issue ...

**Comment by educepter** [comment]
+1 Any update on this one yet?

**Comment by stofancy** [comment]
+ 1 same issue.

**Comment by rodrigoBerlochi** [comment]
+1 here

**Comment by sharanya5** [comment]
@pablosichert Same issue. Any updates on this?

**Comment by UndeadBaneGitHub** [comment]
Still the case, would love an update

**Comment by NwosaEmeka** [comment]
+1 same issue. br tag is injected at random places and using display: none with margin to the span messes up the text
